crates/rethnet_evm/benches/state.rs

```diff
@@ -45,6 +45,100 @@ fn bench_code_by_hash(c: &mut Criterion) {
     );
 }
 
+fn bench_database_commit(c: &mut Criterion) {
+    use hashbrown::HashMap;
+    use revm::primitives::KECCAK_EMPTY;
+
+    use rethnet_eth::remote::optional_u64_from_hex;
+    use rethnet_evm::{Account, AccountInfo, StorageSlot};
+
+    #[derive(Debug, serde::Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    struct AccountState {
+        #[serde(deserialize_with = "optional_u64_from_hex")]
+        nonce: Option<u64>,
+        balance: Option<U256>,
+        storage: HashMap<U256, Option<U256>>,
+        code: Option<Bytes>,
+        storage_cleared: bool,
+    }
+
+    bench_sync_state_method(
+        c,
+        "DatabaseCommit:commit",
+        prep_no_op,
+        |state, _number_of_accounts, _, _| {
+            #[allow(unused_variables)]
+            let mut accounts_to_commit: HashMap<Address, Account> = HashMap::new();
+            let json_accounts: HashMap<Address, AccountState> = serde_json::from_str(
+                &std::fs::read_to_string("benches/accounts_changed_in_mainnet_block_17295357.json")
```

Comment:
Perhaps we should use a directory for this, something like `benches/fixtures`?

Reply:
I like that idea, thank you for suggesting it, done in https://github.com/NomicFoundation/hardhat/pull/3953/commits/250d3d794e7ad656df672f14ad9c56c00b89b5b1